javascriptv3/example_code/kinesis/README.md

```diff
@@ -26,11 +26,6 @@ npm install
 ```
 3. If you're using JavaScript, change the sample file extension from ```.ts``` to ```.js```.
 
+4. To run this code, follow the instructions for this example in the [AWS SDK for JavaScript v3](https://docs.aws.amazon.com/sdk-for-javascript/v3/developer-guide/kinesis-examples-capturing-page-scrolling.html) developer guide. 
```

Comment:
A minor nitpick: I typically make the link text match the title of the destination. That way, when a user gets to the link's target, they can confirm they got to the right target.

Reply:
changed to GitHub profile